modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -1161,6 +1161,64 @@ def resample_var(self, resample_kwargs, ddof, *args, **kwargs):
     def resample_quantile(self, resample_kwargs, q, **kwargs):
         return self._resample_func(resample_kwargs, "quantile", q=q, **kwargs)
 
+    def expanding_aggregate(self, axis, expanding_args, func, *args, **kwargs):
+        new_modin_frame = self._modin_frame.apply_full_axis(
+            axis,
+            lambda df: pandas.DataFrame(
+                df.expanding(*expanding_args).aggregate(func=func, *args, **kwargs)
+            ),
+            new_index=self.index,
```

Comment:
Should we preserve the index when axis=0, and the columns when axis=1?

Reply:
Pandas doesn't support aggregates with `axis=1`.